test/snapshot/testcfg.py

```diff
@@ -0,0 +1,6 @@
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+import testpy
+
+def GetConfiguration(context, root):
+  return testpy.SimpleTestConfiguration(context, root, 'snapshot')
```

Comment:
Is there a specific reason to create a new test configuration for this feature?

Reply:
Just figured that it would be nicer to organize it this way. Since the tests here all need to launch child processes, often times multiple ones in one test, we probably would not benefit much from parallelism.